### ISSUE TITLE
Add ability to ignore files

### ISSUE THREAD

**Issue by kentcdodds** [kiliman/remix-flat-routes#15]
I'd like to colocate `.css` and `.test.tsx` files with my files in my routes. So I'd like to be able to ignore files as routes. Something like this config?

```tsx
module.exports = {
	ignoredRouteFiles: ['**/*'],
	routes: async defineRoutes => {
		return flatRoutes('routes', defineRoutes, {
			ignoredRouteFiles: ['**/.*', '**/*.css', '**/*.test.{js,jsx,ts,tsx}'],
		})
	},
}
```

**Comment by kiliman** [comment]
Are you using flat-files or flat-folders? With folders, it's a non-issue, but with flat-files, I can see why you'd want that option.

Anyway, I will add that option. Sorry it was an oversight.

**Comment by kentcdodds** [comment]
After having read the docs, I see that the folder solution is the intended solution to this kind of colocation which is great 👍 I do think having the option would be good still.

Also, I just saw a reference to `routeIgnorePatterns` in the docs. What is that?

**Comment by kiliman** [comment]
> Also, I just saw a reference to routeIgnorePatterns in the docs. What is that?

Ah, I just copy+pasted from Ryan's gist. I believe he meant `ignoredRouteFiles`. And his commentary was mainly saying how it was difficult to tell which were route files and which weren't without referring to the config. With flat-folders, you know that anything other than *index* (or the various aliases like *_route*, *_layout* etc.) were supporting files.

I'm almost finished with the update. Just writing my tests.

**Comment by kiliman** [comment]
Published as v0.4.4

**Comment by kentcdodds** [comment]
Thank you very much!
